csvcubed/csvcubed/readers/cubeconfig/v1_0/configdeserialiser.py

```diff
@@ -200,7 +211,7 @@ def _configure_remaining_columns_by_convention(
 
             ordered_columns.append(qb_column)
 
-    cube.columns = ordered_columns
+    cube.columns = ordered_columns + list(configured_columns.values())
```

Comment:
didn't we make this change in a different PR that's already been merged into main?

Reply:
No, its not in main,  we did work on this together for this task.